src/wwwroot/controllers/PlanCtrl.js

```diff
@@ -176,12 +177,16 @@
     }
 
     function isValidUpgradeablePlan() {
-      return !requiresDomainConfiguration();
+      return !requiresDomainConfiguration() && !requiresDeliveries();
     }
   
     function requiresDomainConfiguration() {
       return $rootScope.accountLimits.requiresDomainConfiguration || false;
     }
+
+    function requiresDeliveries(){
+      return $rootScope.accountLimits.requiresDeliveries || false;
```

Comment:
the same as !!$rootScope.accountLimits.requiresDeliveries? (not sure)

Reply:
I will check it thanks for the review :)

Reply:
But we need handle this in the planController too because is called first than the auth service (async request take a seconds to response with popupated data). We catch the undefined and convert it to false when the PlanController is initialized. I will use !!$rootScope.accountLimits.requiresDeliveries instead the current sintax. Thanks!